doc/source/conf.py

```diff
@@ -33,7 +33,7 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 root_dir = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
-with open(os.path.join(root_dir, 'VERSION')) as version_file:
+with open(os.path.join(root_dir, 'cobrawap', 'VERSION')) as version_file:
```

Comment:
Coherently, here I would write:
```suggestion
with open(root_dir / 'cobrawap' / 'VERSION') as version_file:
```

Reply:
Implemented.